packages/editor/src/components/post-locked-modal/index.js

```diff
@@ -277,3 +272,13 @@ export default function PostLockedModal() {
 		</Modal>
 	);
 }
+
+/**
+ * A modal component that is displayed when a post is locked for editing by another user.
+ * The modal provides information about the lock status and options to take over or exit the editor.
+ *
+ * @return {React.ReactNode} The rendered PostLockedModal component.
+ */
+export default globalThis.IS_GUTENBERG_PLUGIN
+	? withFilters( 'editor.PostLockedModal' )( PostLockedModal )
+	: PostLockedModal;
```

Comment:
We could also use a Symbol for a private filter, but this way we avoid unnecessarily wrapping the component.

```js
const HOOK_NAME = globalThis.IS_GUTENBERG_PLUGIN
	? 'editor.PostLockedModal'
	: Symbol( 'editor.PostLockedModal' );
```